hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/SamlEngineConfiguration.java

```diff
@@ -100,6 +100,11 @@ protected SamlEngineConfiguration() {}
     @JsonProperty
     protected URI configUri;
 
+    @Valid
+    @NotNull
+    @JsonProperty
+    protected Duration certificatesConfigCacheExpiry;
```

Comment:
Might be nice to have a default value here.

Reply:
agree, we'll not need this PR then https://github.com/alphagov/ida-hub-acceptance-tests/pull/159

Reply:
made the change